docs/TroubleshootingGuide.md

```diff
@@ -411,28 +411,35 @@ There are certain errors that likely point to the eBPF client. These errors will
 - `ERROR_FILE_NOT_FOUND`. This indicates that the application tried to open an eBPF program with an invalid path. This
   points to an issue within the application. The resolution is to change the path used by the application.
 
+- `Program type is not valid`. The trace shows `Program type` guid is zero. Hence `ebpf_program_create` failed. The subsequent traces show 'An invalid parameter was passed to a service or function' and 'The parameter is incorrect' indicating that the user-mode application failed to set a valid program type in the eBPF program.
```

Comment:
The other errors shown above are the error code values. Do we want to put ERROR_INVALID_PARAMETER here instead?

Reply:
Added. Thanks.